packages/shared/src/components/profile/UserList.tsx

```diff
@@ -31,6 +53,29 @@ function UserList({
         <UserShortInfoPlaceholder placeholderAmount={placeholderAmount} />
       }
     >
+      {!!squad && (
```

Comment:
Thinking about it, maybe we can simply add a "initialItem" as it's a very generic component and we now narrow it down with a squad specific feature.

Reply:
Also thought about this. It can be done for sure.